.php-cs-fixer.php

```diff
@@ -93,9 +94,268 @@
+        'linebreak_after_opening_tag' => true,
+        // Magic constants should be referred to using the correct casing.
+        'magic_constant_casing' => true,
+        // Magic method definitions and calls must be using the correct casing.
+        'magic_method_casing' => true,
+        // Method chaining MUST be properly indented. Method chaining with different levels of indentation is not supported.
+        'method_chaining_indentation' => true,
+        // DocBlocks must start with two asterisks, multiline comments must start with a single asterisk, after the opening slash. Both must end with a single asterisk before the closing slash.
+        'multiline_comment_opening_closing' => true,
+        // Forbid multi-line whitespace before the closing semicolon or move the semicolon to the new line for chained calls.
+        'multiline_whitespace_before_semicolons' => false,
+        // Function defined by PHP should be called using the correct casing.
+        'native_function_casing' => true,
+        // Native type declarations should be used in the correct case.
+        'native_type_declaration_casing' => true,
+        // Master language constructs shall be used instead of aliases.
+        'no_alias_language_construct_call' => true,
+        // Replace control structure alternative syntax to use braces.
+        'no_alternative_syntax' => true,
+        // There should not be a binary flag before strings.
+        'no_binary_string' => true,
+        // There should not be blank lines between docblock and the documented element.
+        'no_blank_lines_after_phpdoc' => false,
+        // There should not be any empty comments.
+        'no_empty_comment' => true,
+        // Remove useless (semicolon) statements.
+        'no_empty_statement' => true,
+        // Removes extra blank lines and/or blank lines following configuration.
+        'no_extra_blank_lines' => [
+            'tokens' => [
+                'attribute',
+                'break',
+                'case',
+                'continue',
+                'curly_brace_block',
+                'default',
+                'extra',
+                'parenthesis_brace_block',
+                'return',
+                'square_brace_block',
+                'switch',
+                'throw',
+                'use'
+            ]
+        ],
+        // The namespace declaration line shouldn't contain leading whitespace.
+        'no_leading_namespace_whitespace' => true,
+        // Either language construct `print` or `echo` should be used.
+        'no_mixed_echo_print' => true,
+        // Operator `=>` should not be surrounded by multi-line whitespaces.
+        'no_multiline_whitespace_around_double_arrow' => true,
+        // Properties MUST not be explicitly initialized with `null` except when they have a type declaration (PHP 7.4).
+        'no_null_property_initialization' => true, // TODO Arguable, could remove info about nullability
```

Comment:
Lets disable this for now, we can revisit it once we have native types.
```suggestion
        'no_null_property_initialization' => false, // TODO Arguable, could remove info about nullability
```